src/main/java/bio/terra/janitor/app/StartupInitializer.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.janitor.app;
+
+import bio.terra.janitor.app.configuration.JanitorJdbcConfiguration;
+import bio.terra.janitor.service.migirate.MigrateService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+
+public final class StartupInitializer {
```

Comment:
I'd appreciate a comment on what this class is for and how/when it should be used in spring configuration.

Reply:
added